distribution/bin/export.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+checkCmd() {
+  if ! command -v wget > /dev/null 2>&1; then
+    echo -e '[ERROR] This system does not contains wget command, please install it!'
+    exit 1
+  fi
+}
+
+download() {
+  if [[ -e /tmp/rocketmq/ ]]; then
+    rm -rf /tmp/rocketmq/
+  fi
+
+  wget --no-check-certificate -P /tmp/rocketmq/ https://ons-migration.oss-cn-hangzhou.aliyuncs.com/rocketmq-for-export.tar.gz
+  if [[ $? -ne 0 ]]; then
+    echo -e "[ERROR] Download rocketmq error, please make sure this file exists"
+    exit 1
+  else
+    echo -e "[INFO] Download rocketmq completed，file path: $(pwd)/rocketmq-for-export.tar.gz"
+  fi
+
+  tar -zxvf /tmp/rocketmq/rocketmq-for-export.tar.gz -C /tmp/rocketmq/ > /dev/null 2>&1
+
+  echo -e "[INFO] Unzip rocketmq-for-export.tar.gz completed"
+}
```

Comment:
It is not appropriate to download packages from OSS here

Reply:
Any update for here?

Reply:
has been deleted about oss